web/src/lib/static/admin/findDeadLinks.ts

```diff
@@ -238,29 +265,42 @@ export const findDeadLinks = async (): Promise<Record<string, string[]>> => {
     );
   };
 
-  for (const page of pages) {
-    const promise = new Promise((resolve) => {
-      const htmlUrlChecker = new HtmlUrlChecker(
-        {},
-        {
-          link: (result: any): void => {
-            if (result.broken && !isTemplateLink(result.url.original)) {
-              deadLinks[page].push(result.url.original);
-            }
-          },
-          end: (): void => {
-            resolve({});
-          },
-        }
-      );
-      const url = new URL(process.env.REDIRECT_URL || "");
-      htmlUrlChecker.enqueue(`${url.origin}${page}`, {});
-    });
+  const numberOfBatches = Math.ceil(pages.length / 5);
+
+  for (let j = 0; j < numberOfBatches; j++) {
+    const pagePromises = [];
+    const startIndex = j * 5;
+    const batch = pages.slice(startIndex, startIndex + 5);
+
+    for (const page of batch) {
+      const promise = new Promise((resolve) => {
+        const htmlUrlChecker = new HtmlUrlChecker(
+          {},
+          {
+            link: (result: any): void => {
+              if (
+                result.broken &&
+                !isTemplateLink(result.url.original) &&
+                result.url.original !== "https://twitter.com/i/lists/1148625562895826945"
```

Comment:
So the way twitter works now, is that you need an account to access any page without a 403, so we run into this issue where the page just has this link for every page and it ended up taking much longer, so I've filtered it out.

Reply:
Twitter is only ever mentioned in our footer, correct?

I'm comfortable with this.

Reply:
Yes, thats my understanding, in the footer on every page.